### ISSUE TITLE
fix: Missing some locale codes in note

### ISSUE THREAD

**Issue by StefSmit12** [typestack/class-validator#1043]
### Description

ispostalcode can validate postcodes more countries than stated in the notes. By example; note is missing the country codes SG, SI, SK.

**Minimal code-snippet showcasing the problem**
 

```ts
/**
 * Check if the string is a postal code,
 * (locale is one of [ 'AD', 'AT', 'AU', 'AZ', 'BE', 'BG', 'BR', 'CA', 'CH', 'CZ', 'DE', 'DK', 'DO', 'DZ', 'EE', 'ES', 'FI', 'FR', 'GB', 'GR', 'HR', 'HU', 'ID', 'IE' 'IL', 'IN', 'IR', 'IS', 'IT', 'JP', 'KE', 'KR', 'LI', 'LT', 'LU', 'LV', 'MX', 'MT', 'MY', 'NL', 'NO', 'NP', 'NZ', 'PL', 'PR', 'PT', 'RO', 'RU', 'SA', 'SE', 'SG', 'SI', 'SK', 'TN', 'TW', 'UA', 'US', 'ZA', 'ZM' ] OR 'any'. If 'any' is used, function will check if any of the locals match. Locale list is validator.isPostalCodeLocales.).
 * If given value is not a string, then it returns false.
 */
```

**Comment by NoNameProvided** [comment]
I have removed the hard-coded JSDoc list. The function is typed so users will always see the up-to-date list of options as code completion. 

Landed in https://github.com/typestack/class-validator/commit/cf8d2c0602faef82d8964a8a6718242a945dd8db.

**Comment by NoNameProvided** [comment]
This feature/fix has been released in `0.14.0`.
